packages/ui-kit/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@propeldata/ui-kit",
-  "version": "0.3.0-rc.3",
+  "version": "0.4.0",
   "license": "MIT",
   "source": "src/index.ts",
   "main": "dist/cjs/index.js",
-  "module": "dist/mjs/index.module.js",
+  "module": "dist/esm/index.module.js",
   "scripts": {
-    "build": "rm -rf dist && yarn graphql:build && tsc -p tsconfig.mjs.json && tsc -p tsconfig.cjs.json",
+    "build": "rm -rf dist && yarn graphql:build && rollup -c",
```

Comment:
It does with `rollup-plugin-typescript2` plugin (we have it), but since the plugin is mostly focused on transpilation rather than type checking, I've included `tsc --noEmit` into the build script. Also, I've enabled `strict` in tsconfig and fixed most of the complaints.

Reply:
Awesome!